### ISSUE TITLE
OnePlus2: Android 5.1.1 OxygenOS 2.1.2 (newinstaller-fj)

### ISSUE THREAD

**Issue by TheBlackRabbit** [offensive-security/kali-nethunter#270]
OnePlus2 Android 5.1.1 OxygenOS 2.1.2
--
This morning, I received an OTA update notification. OxygenOS 2.1.1 > 2.1.3 I'm starting this to see what if any problems might exist with nethunter builds produced specifically from the "newinstaller-fj" kali-nethunter branch.

Cloning the latest newinstaller-fj as I know @binkybear has made a TON of revisions to the repo in the past few days. Will post any issues and shots and request for screens to help debug any open issues.

[ Stock OS/Kernel - OxygenOS 2.1.2 ]

![screenshot_2015-11-16-11-25-10](https://cloud.githubusercontent.com/assets/13502768/11194051/d2cd5fa6-8c5e-11e5-86fc-09d5120d0f53.png)
-
**Current Issues**

- [ ] still having issues scanning/connecting/bringing up wlan0 interface after flashing
- [x] ***fixed*** aroma stalls out at 40%. ( issued adb shell and passed killall5 to get out without forceing reboot)
          *Reference upcoming posts for more detail on each specific issue*

*Note:* this list will only cover a couple general issues that would be needed to get a base kali install w/ standard system funtionality running; any other issues will be posted as seperate issues...

**Comment by TheBlackRabbit** [comment]
@binkybear  [11/18/2015] 

**UPDATE**
- [x] ***fixed*** aroma stalls out at 40%
- [ ] wlan0 cannot connect or scan APs (*seems to be kernel related, partially working. update pending further investgation*)

**Comment by TheBlackRabbit** [comment]
@binkybear - I flashed the kernel seperately after a successful aroma install that for some reason couldnt install the kernel... However wifi worked until after i flashed the kernel seperately. I just finished flashing a clean 2.1.1 install with the newst build and the same issue is occurring as well with wifi.

wlan0 can be brought up down to no availe, cannot scan APs (ANY) or connect to saved networks. wlan0's interface assignment is given to usb-c wlan adapter (wn722n and ra)
